Kudu.Core/SiteExtensions/SiteExtensionInstallationLock.cs

```diff
@@ -5,6 +5,8 @@ namespace Kudu.Core.SiteExtensions
 {
     public class SiteExtensionInstallationLock : LockFile
     {
+        private const string LockNameSuffix = "install.lock";
```

Comment:
correct! each extension will has it own folder. if two request try to install the exact same extension at the same time. we let one in, and reject (409 conflict) the other one.